gst/nnstreamer/tensor_query/tensor_query_client.c

```diff
@@ -502,45 +489,38 @@ gst_tensor_query_client_sink_event (GstPad * pad,
   switch (GST_EVENT_TYPE (event)) {
     case GST_EVENT_CAPS:
     {
-      GstCaps *caps;
-      GstStructure *structure;
-
-      gst_event_parse_caps (event, &caps);
-      structure = gst_caps_get_structure (caps, 0);
-      gst_tensors_config_from_structure (&self->in_config, structure);
+      gst_event_parse_caps (event, &self->in_caps);
```

Comment:
plz copy caps in event.
gst document says that, the caps from gst_event_parse_caps() is available until the event is valid.

Reply:
Or, I think it is ok to get caps string, not gst-caps instance.

```
GstCaps *caps;

gst_event_parse_caps (event, &caps);
.....

if (ret) {
  g_free (self->current_caps);
  self->current_caps = gst_caps_string (caps);
}

return ret;
```